lib/step.rb

```diff
@@ -212,8 +215,8 @@ def run(dry_run: false, fail_fast: false)
         next if path.blank?
 
         # GitHub Actions has a 64KB maximum for annotiations. That's a bit
-        # too long so instead let's go for a maximum of 24KB or 256 lines.
-        annotation_output = truncate_output(@output, max_kb: 24, max_lines: 256)
+        # too long so instead let's go for a maximum of 256 lines.
+        annotation_output = truncate_output(@output, max_lines: 256, context_lines: 10)
```

Comment:
would be good to keep `max_kb` if possible to handle the cases where there's extremely long lines (multiple KB).

Reply:
Done!